plugins/org.jboss.tools.quarkus.lsp4e/src/org/jboss/tools/quarkus/lsp4e/internal/handlers/qute/GenerateTemplateFileHandler.java

```diff
@@ -0,0 +1,97 @@
+package org.jboss.tools.quarkus.lsp4e.internal.handlers.qute;
+
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+import java.util.Map;
+
+import org.eclipse.core.commands.ExecutionEvent;
+import org.eclipse.core.commands.ExecutionException;
+import org.eclipse.core.resources.IFile;
+import org.eclipse.core.resources.IResource;
+import org.eclipse.core.resources.ResourcesPlugin;
+import org.eclipse.core.runtime.CoreException;
+import org.eclipse.core.runtime.IPath;
+import org.eclipse.core.runtime.IStatus;
+import org.eclipse.core.runtime.NullProgressMonitor;
+import org.eclipse.core.runtime.Status;
+import org.eclipse.jdt.annotation.NonNull;
+import org.eclipse.lsp4e.LanguageServiceAccessor;
+import org.eclipse.lsp4e.command.LSPCommandHandler;
+import org.eclipse.lsp4j.Command;
+import org.eclipse.lsp4j.ExecuteCommandOptions;
+import org.eclipse.lsp4j.ExecuteCommandParams;
+import org.eclipse.lsp4j.services.LanguageServer;
+import org.eclipse.ui.IWorkbenchWindow;
+import org.eclipse.ui.PlatformUI;
+import org.jboss.tools.quarkus.lsp4e.QuarkusLSPPlugin;
+
+public class GenerateTemplateFileHandler extends LSPCommandHandler {
+
+	private static final String TEMPLATE_FILE_URI = "templateFileUri"; //$NON-NLS-1$
+	private static final String QUTE_COMMAND_GENERATE_TEMPLATE_CONTENT = "qute.command.generate.template.content"; //$NON-NLS-1$
```

Comment:
Qute LS should provide a QuteServerCommandConstants to avoid hard conding that here.

Reply:
the only reference i can find is QuteGenerateTemplateContentCommandHandler.COMMAND_ID

Reply:
ok then, reverting and i'm creating an issue on Qute side for adding this constant in the future release